src/main/java/core/selenium/TrabajopolisConfig.java

```diff
@@ -62,8 +63,9 @@ public Properties config() {
             properties.load(input);
             username = properties.getProperty("username");
             password = properties.getProperty("password");
-            url = properties.getProperty("url");
-            urlAccount = properties.getProperty("urlAccount");
+            urlLogin = properties.getProperty("urlLogin");
+            urlProfile = properties.getProperty("urlProfile");
+            urlBase = properties.getProperty("urlBase");
         } catch (Exception e) {
             e.printStackTrace();
```

Comment:
stop de execution please use a throw to send a null pointer exception.

Reply:
Done. 
I need an Exception because the method load at line 65 needs for work, for solve this question i put two catch with different exceptions is my solution.

Reply:
I am sorry, but I want to say that you throw n exception E.g.
`catch (FileNotFoundException e) {
            e.printStackTrace();
           throw new NullPointerException("This file not exist!!", e)
        }
`
With this throw your app will be stop

Reply:
done